From 0b97c4b07830c0873f0d863ee8a63b294f0112e4 Mon Sep 17 00:00:00 2001
From: Cary Coutant <ccoutant@gmail.com>
Date: Fri, 19 Mar 2021 15:29:49 -0700
Subject: [PATCH] Fix typo in previous patch: should use struct mallinfo2.

	PR gold/26585
	* main.cc (main): Fix typo in previous patch.

Upstream-Status: Backport [cc1849716f5 Fix typo in previous patch: should use struct mallinfo2.]

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 gold/main.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gold/main.cc b/gold/main.cc
index 89bc40b22af..ff44cbb7add 100644
--- a/gold/main.cc
+++ b/gold/main.cc
@@ -291,7 +291,7 @@ main(int argc, char** argv)
               elapsed.wall / 1000, (elapsed.wall % 1000) * 1000);
 
 #if defined(HAVE_MALLINFO2)
-      struct mallinfo m = mallinfo2();
+      struct mallinfo2 m = mallinfo2();
       fprintf(stderr, _("%s: total space allocated by malloc: %lld bytes\n"),
 	      program_name, static_cast<long long>(m.arena));
 #elif defined(HAVE_MALLINFO)
